core/ledger/kvledger/kv_ledger.go

```diff
@@ -64,6 +64,9 @@ type kvLedger struct {
 	// reconciliation and may be updated during a regular block commit.
 	// Hence, we use atomic value to ensure consistent read.
 	isPvtstoreAheadOfBlkstore atomic.Value
+
+	commitNotifierLock sync.RWMutex
```

Comment:
You're using an `RWMutex` but I don't see any calls to `RLock`. Should this be a simple `sync.Mutex`?

Reply:
Done